Sources/StreamChatUI/ChatChannelList/ChatChannelListVC+SwiftUI.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import SwiftUI
+
+@available(iOS 13.0, *)
+/// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
+public typealias ChatChannelList = ChatChannelListVC._ChatChannelList<NoExtraData>
+
+extension _ChatChannelListVC {
+    /// A `UIViewControllerRepresentable` subclass which wraps `ChatChannelListVC` and shows list of channels.
+    public struct _ChatChannelList<ExtraData: ExtraDataTypes>: UIViewControllerRepresentable {
+        /// The `ChatChannelListController` instance that provides channels data.
+        let controller: _ChatChannelListController<ExtraData>
```

Comment:
all mentions of `<ExtraData>` here are IMO redundant. `<ExtraData>` type will be provided by the enclosing `_ChatChannelListVC`

Reply:
Are we actually sure about this? IMO the `ExtraData` should be defined also for the controller, otherwise we would always initialize `ChatChannelListVC` with `NoExtraData` in `makeUIViewController`. Will it suffice?

Reply:
@IgorRosocha yes, sorry, the controller needs it, the VC mentions shouldn't need iit